account_setup.py

```diff
@@ -0,0 +1,369 @@
+    subscription_client = SubscriptionClient(creds)
+    tenant_ids = [tenant.id for tenant in subscription_client.tenants.list()]
+    if len(tenant_ids) != 1:
+        tenant_id = prompt_tenant_selection(tenant_ids)
+    else:
+        tenant_id = tenant_ids[0]
+
+    print("Input the desired names and values for your Azure resources. "\
+          "Default values are provided in the brackets. "\
+          "Hit enter to use default.")
+    kwargs = {
+        "region": prompt_with_default("Azure Region", DefaultSettings.region),
+        "resource_group": prompt_with_default("Resource Group Name", DefaultSettings.resource_group),
+        "storage_account": prompt_with_default("Storage Account Name", DefaultSettings.storage_account),
+        "batch_account": prompt_with_default("Batch Account Name", DefaultSettings.batch_account),
+        # "virtual_network_name": prompt_with_default("Virtual Network Name", DefaultSettings.virtual_network_name),
+        # "subnet_name": prompt_with_default("Subnet Name", DefaultSettings.subnet_name),
+        "application_name": prompt_with_default("Active Directory Application Name", DefaultSettings.application_name),
+        "application_credential_name": prompt_with_default("Active Directory Application Credential Name", DefaultSettings.resource_group),
+        "service_principal": prompt_with_default("Service Principal Name", DefaultSettings.service_principal)
+    }
+    print("Creating the Azure resources.")
+
+    # create resource group
+    resource_group_id = create_resource_group(creds, subscription_id, **kwargs)
+    kwargs["resource_group_id"] = resource_group_id
+
+    # create storage account
+    storage_account_id = create_storage_account(creds, subscription_id, **kwargs)
+    kwargs["storage_account_id"] = storage_account_id
+
+    # create batch account
+    batch_account_id = create_batch_account(creds, subscription_id, **kwargs)
+
+    # create vnet with a subnet
+    # subnet_id = create_vnet(creds, subscription_id)
+
+    # create AAD application and service principal
+    profile = credentials.get_cli_profile()
+    aad_cred, subscirption_id, tenant_id = profile.get_login_credentials(
+        resource=AZURE_PUBLIC_CLOUD.endpoints.active_directory_graph_resource_id
+    )
+
+    application_id, service_principal_object_id, application_credential = create_aad_user(aad_cred, tenant_id)
+
+    create_role_assignment(creds, subscription_id, resource_group_id, service_principal_object_id)
+
+    secrets = format_secrets(
+        **{
+            "tenant_id": tenant_id,
+            "client_id": application_id,
+            "credential": application_credential,
+            # "subnet_id": subnet_id,
+            "batch_account_resource_id": batch_account_id,
+            "storage_account_resource_id": storage_account_id
+        }
+    )
+
+    print("\n# Copy the following into your .aztk/secrets.yaml file\n", secrets)
```

Comment:
This is still printing with a space. I believe this will work
```python
 print("\n# Copy the following into your .aztk/secrets.yaml file\n{}".format(secrets))
```